app/wasm/test/custom_query_test.go

```diff
@@ -43,6 +43,27 @@ func SetupCustomApp(t *testing.T, addr sdk.AccAddress) (*app.OsmosisApp, sdk.Con
 	return osmosis, ctx
 }
 
+func TestQueryFullDenom(t *testing.T) {
+	actor := RandomAccountAddress()
+	osmosis, ctx := SetupCustomApp(t, actor)
+
+	reflect := instantiateReflectContract(t, ctx, osmosis, actor)
+	require.NotEmpty(t, reflect)
+
+	// query full denom
+	query := wasmbindings.OsmosisQuery{
+		FullDenom: &wasmbindings.FullDenom{
+			Contract: actor.String(),
+			SubDenom: "ustart",
+		},
+	}
+	resp := wasmbindings.FullDenomResponse{}
+	queryCustom(t, ctx, osmosis, reflect, query, &resp)
+
+	expected := fmt.Sprintf("cw/%s/ustart", actor.String())
+	require.EqualValues(t, expected, resp.Denom)
+}
+
 func TestQueryPool(t *testing.T) {
```

Comment:
Comment on the file below... you didn't use the download script?
Please run it, it will also update version.txt file

Reply:
Forgot its sorry. Now fixed.